docs/delivery/README.md

```diff
@@ -0,0 +1,121 @@
+
+## Day 1 Proposal
+
+### Error Sink
+
+Channels are responsible for forwarding received messages to subscribers. When they fail to do so, they are responsible for sending the failing messages to an **error sink**. It could be a channel but it does not have to.
+
+Similarly Event Sources are responsible sending events to a sink and when they fail to do so, they are responsible for sending the failing events to an **error sink**.
+
+### Dead-Letter Channel
+
+Channel implementations might leverage the existing native error handling support they provide, usually a dead letter channel, to forward failed messages to the error sink. In that case, the error sink might be realized by creating a subscription on the error channel.
+
+### Retry
+
+Typically channel implementations and event sources retry sending messages before redirecting them to the error sink.
+While there are many different ways to implement the retry logic
+(immediate retry, retry queue, etc...), implementations usually
+rely on a common set of configuration parameters, such as
+the number of retries and the interval between retries.
+
+### Delivery Specification
+
+The goal of this delivery specification is to formally define the vocabulary related to capabilities defined above (error sink, dead-letter queues and retry) to provide consistency across all Knative event sources, channels and brokers.
+
+The minimal delivery specification looks like this:
+
+```go
+
+// DeliverySpec contains the delivery options for event senders,
+// such as channelable and source.
+type DeliverySpec struct {
+	// ErrorSink is the sink receiving messages that couldn't be sent to
+	// a destination.
+	// +optional
+	ErrorSink *apisv1alpha1.Destination `json:"errorSink,omitempty"`
+
+	// Retry is the minimum number of retries the sender should attempt when
+	// sending a message before moving it to the error sink.
+	// +optional
+	Retry *int32 `json:"retry,omitempty"`
+
+	// BackoffPolicy is the retry backoff policy (linear, exponential)
+	// +optional
+	BackoffPolicy *BackoffPolicyType `json:"backoffPolicy,omitempty"`
+
+	// BackoffDelay is the delay before retrying.
+	// More information on Duration format: https://www.ietf.org/rfc/rfc3339.txt
+	//
+	// For linear policy, backoff delay is the time interval between retries.
+	// For exponential policy , backoff delay is backoffDelay*10^<numberOfRetries>
+	// +optional
+	BackoffDelay *string
+}
+
+// BackoffPolicyType is the type for backoff policies
+type BackoffPolicyType string
+
+const (
+	// Linear backoff policy
+	BackoffPolicyLinear BackoffPolicyType = "linear"
+
+	// Exponential backoff policy
+	BackoffPolicyExponential BackoffPolicyType = "exponential"
+)
+```
+
+Channel, brokers and event sources  are not required to support all this capabilities and are free to add more delivery options.
+
+### Exposing underlying DLC
+
+Channels supporting dead letter queue should advertise it in their status.
```

Comment:
we mix wording here. DLC / DLQ 

EIP calls it "Dead Letter Channel", let's stay with that ?